src/pages/terms-and-conditions/_business-grid.tsx

```diff
@@ -78,7 +85,9 @@ const BusinessGrid = () => {
                     title="_t_General terms of use_t_"
                     content="_t_Terms and ethical standards for all our affiliates, introducing brokers, API users, and payment agents_t_"
                     url={
-                        is_row
+                        is_row && language === 'es'
```

Comment:
What if a user is from EU and selects Spanish? Then they will get this file _/tnc/business-partners-general-terms-eu.pdf_ but shouldn't they get /tnc/business-partners-general-terms-**es**.pdf?

Reply:
No as of now they have not yet done the EU for spanish..so we will show the EU pdf..once that's done they will update us